FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -181,6 +181,7 @@ private enum GoogleDataTransportConfig {
         self.settings.updateSettings()
 
         self.addSubscriberFields(event: event)
+        event.setSamplingRate(samplingRate: self.settings.samplingRate)
```

Comment:
Did we miss this field in the event earlier? Should I add this field to the PLX alert?

Reply:
Yeah we somehow made this method but never called it - worth adding to plx alerts